src/pages/home/report/ReportActionCompose.js

```diff
@@ -161,7 +170,7 @@ class ReportActionCompose extends React.Component {
         if (this.textInput) {
             // There could be other animations running while we trigger manual focus.
             // This prevents focus from making those animations janky.
```

Comment:
Please elaborate on why this is helpful. Which "interaction" are we waiting to finish ?

Reply:
So When the modal screen is being closed, we want to wait for those before focusing on it. And I noticed that manual focus is causing a lot of issues so I think it's better to put focus after Interactions, we have no side effects.

Reply:
Is there some way to see which "interactions" we are waiting for? Or am I misunderstanding what this method does? Which janky animation are we fixing specifically with this? Is there a before/after video we can share to verify the change does what it should and so QA can verify the change as well?

I'm not seeing anything at all about the emoji picker in the test steps so those will also need to be updated.

Reply:
+1 @parasharrajat Can you updated the test steps to include something about the emoji picker? It's not super apparent that it's a modal and it might get missed in QA.

Reply:
Sure @jasperhuangg